src/main/java/uk/gov/gchq/magmacore/database/MagmaCoreRemoteSparqlDatabase.java

```diff
@@ -55,39 +55,70 @@ public class MagmaCoreRemoteSparqlDatabase implements MagmaCoreDatabase {
     private final RDFConnection connection;
 
     /**
-     * Constructor to create a connection to a SPARQL endpoint.
+     * Constructs a MagmaCoreRemoteSparqlDatabase connection to a SPARQL server.
      *
-     * @param serviceUrl the URL String of the SPARQL update endpoint
+     * @param serviceUrl The URL of the SPARQL update endpoint.
      */
     public MagmaCoreRemoteSparqlDatabase(final String serviceUrl) {
-      connection = RDFConnectionRemote.newBuilder()
-       .destination(serviceUrl)
-       .queryEndpoint("query")
-       .updateEndpoint("update")
-       .triplesFormat(RDFFormat.RDFJSON)
-       .build();
+        connection = RDFConnectionRemote.newBuilder().destination(serviceUrl).queryEndpoint("query")
```

Comment:
The old formatting was easier to read and is more common for chained methods.

Reply:
This applies throughout so I won't repeat it everywhere.

Reply:
I concur - However, I've yet to be find a java-formatter setting that auto-formats lines like this. Do you know how to get it to wrap chained method calls?

Reply:
The simplest is probably to set `never wrap already wrapped lines` which will retain manual formatting, or to use the `formatter:on` and `formatter:off` tags where needed. Otherwise it looks like a minefield of options.

Reply:
I've disabled the rule `org.eclipse.jdt.core.formatter.join_wrapped_lines` in the java-formatter.xml and reverted the affected lines.